### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MarcoBrunoBR**
**CHANGELOG** :memo:

*

Added Quick Start

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # gaiden
 GetNinjas Styleguide
+
+# Quick Start
+- Clone the repository: `git clone https://github.com/getninjas/gaiden.git`
+- Configure the build links with: `npm run config`
```
Comment (eduardojmatos) [inline README.md]: @MarcoBrunoBR can you insert the `npm install` too?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+- Run project with: `npm run dev`
```
Comment (eduardojmatos) [inline README.md]: @MarcoBrunoBR it's necessary to describe which port we are using (`8001` for serve `build/*` files), including the URL of docs ([http://localhost:8000/gaiden-css/index.html](http://localhost:8000/gaiden-css/index.html)).

**Comment by MarcoBrunoBR** [comment]
https://github.com/getninjas/gaiden/pull/31

**Comment by MarcoBrunoBR** [comment]
https://github.com/getninjas/gaiden/pull/32

**Pull request closed without merging**